persistence.js

```diff
@@ -33,6 +31,8 @@ function RedisPersistence (opts) {
   this.maxSessionDelivery = opts.maxSessionDelivery || 1000
   this._db = new Redis(opts)
 
+  this.messageIdCache = {}
```

Comment:
Isn't a es6 Map heavier cpu/memory wise? what about a LRU module like you mentioned before?